Libraries/LibWeb/HTML/TraversableNavigable.h

```diff
@@ -142,6 +146,10 @@ class TraversableNavigable final : public Navigable {
     // https://html.spec.whatwg.org/multipage/document-sequences.html#system-visibility-state
     VisibilityState m_system_visibility_state { VisibilityState::Hidden };
 
+    // https://storage.spec.whatwg.org/#traversable-navigable-storage-shed
+    // A traversable navigable holds a storage shed, which is a storage shed. A traversable navigable’s storage shed holds all session storage data.
```

Comment:
(not sure if citing the first sentence helps with understanding anything.)

Reply:
I find it useful